integration-tests/build.gradle.kts

```diff
@@ -49,3 +49,15 @@ dependencies {
 }
 
 tasks { bootJar { enabled = false } }
+
+apply(from = "$rootDir/scripts.gradle.kts")
+
+@Suppress("UNCHECKED_CAST")
+val enableTestConcurrency = extra["enableTestConcurrency"] as (Test) -> Unit
+
+tasks.test {
+  enableTestConcurrency(this)
```

Comment:
Can we also set `setForkEvery(1)`?

Reply:
We are running in concurrency. Do we need that?

Reply:
Hm... actually, I don't think we do anymore since we start the servers before we start running tests.